middleware/settings.py

```diff
@@ -192,35 +197,36 @@
 
 CELERY_BEAT_SCHEDULER = "django_celery_beat.schedulers:DatabaseScheduler"
 
+
+ENABLE_UTC = True
+
 HOST_NAME = env("HOST_NAME")
-CSRF_TRUSTED_ORIGINS = ["https://care-dev-middleware.10bedicu.in"]
+
+CSRF_TRUSTED_ORIGINS = env.list("CSRF_TRUSTED_ORIGINS")
 
 # Observations
-REDIS_OBSERVATIONS_KEY = env("REDIS_OBSERVATIONS_KEY")
+REDIS_OBSERVATIONS_KEY = "observations"
 UPDATE_INTERVAL = env.int("UPDATE_INTERVAL", default=60)
 
 
 # Cameras
-WSDL_PATH = env("WSDL_PATH")
-HOSTNAME = env("HOSTNAME")
-ONVIF_USERNAME = env("ONVIF_USERNAME")
-PASSWORD = env("PASSWORD")
-PORT = env("PORT")
+WSDL_PATH = "/venv/lib/python3.12/site-packages/wsdl/"
```

Comment:
https://github.com/ohcnetwork/teleicu_middleware_backend/pull/1#discussion_r1740155422

Reply:
have added it